src/OpenToolkit.Core/Loader/IPlatformLibraryNameContainer.cs

```diff
@@ -0,0 +1,39 @@
+namespace OpenToolkit.Core.Loader
+{
+    /// <summary>
+    /// Defines a class that holds library names used on various platforms.
+    /// </summary>
+    public interface IPlatformLibraryNameContainer
```

Comment:
Couldn't we just make this a single function of `OSPlatform -> String`? Seems like it would be very easy to write as a switch statement.

Reply:
IDK, this was a file that was previously existent in the library, i removed it when i removed all ADL references from the core project. Now two of them have been re-added as they were essential to openAL

Reply:
No problem. If we do re-add it, then we should try to rework it into a simple interface with a function instead.